UI/i18n/Translations.ts

```diff
@@ -910,74 +981,142 @@ export default class Translations {
                     fr: "<h3>Plus de thème </h3>Vous aimez collecter des données? <br/>Il y a plus de thèmes disponible.",
                     nl: "<h3>Meer thema's</h3>Vind je het leuk om geodata te verzamelen? <br/> Hier vind je meer opties."
                 }),
                 
                 requestATheme: new T({
                     en: "If you want a custom-built quest, request it <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>here</a>",
-                    nl: "Wil je een eigen kaartthema, vraag dit <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>hier aan</a>"
+					ca: "Si vols que et fem una petició pròpia , demana-la <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>aquí</a>",
+					es: "Si quieres que te hagamos una petición propia , pídela <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>aquí</a>",
+                    nl: "Wil je een eigen kaartthema, vraag dit <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>hier aan</a>",
+                    fr: "Si vous voulez une autre carte thématique, demandez <a href='https://github.com/pietervdvn/MapComplete/issues' target='_blank'>ici</a>"
                 }),
 
                 streetcomplete: new T({
                     en: "Another, similar application is <a href='https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete' target='_blank'>StreetComplete</a>",
+					ca: "Una altra aplicació similar és <a href='https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete' target='_blank'>StreetComplete</a>",
+					es: "Otra aplicación similar es <a href='https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete' target='_blank'>StreetComplete</a>",
                     fr: "Une autre application similaire est <a href='https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete' target='_blank'>StreetComplete</a>",
                     nl: "Een andere, gelijkaardige Android-applicatie is <a href='https://play.google.com/store/apps/details?id=de.westnordost.streetcomplete' target='_blank'>StreetComplete</a>"
+                }),
+                createYourOwnTheme: new T({
+                    en: "Create your own MapComplete theme from scratch",
+					ca: "Crea la teva pròpia petició completa de MapComplete des de zero.",
+					ca: "Crea tu propia petición completa de MapComplete desde cero.",
+                    nl: "Maak je eigen MapComplete-kaart",
+                    fr: "Créez votre propre MapComplete carte"
                 })
             },
             readYourMessages: new T({
                 en: "Please, read all your OpenStreetMap-messages before adding a new point.",
+				ca: "Llegeix tots els teus missatges d\'OpenStreetMap abans d\'afegir nous punts.",
+				ca: "Lee todos tus mensajes de OpenStreetMap antes de añadir nuevos puntos.",
```

Comment:
Here, you marked both of the texts to be catalan. WIth others I could figure out which one was which, but here I cant...

Reply:
fixed also